.travis.yml

```diff
@@ -17,6 +17,8 @@ matrix:
         - php: nightly
 
 before_install:
+    - source .travis/xdebug.sh
```

Comment:
I disagree with this approach: while it might shorten some builds, it is also true that it adds a lot of coupling with travis' infrastructure, which may change at any time.

Instead, let's limit ourselves to running coverage conditionally, please.

Reply:
Adjusted!